gulpfile.js

```diff
@@ -0,0 +1,127 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview node module that automate tasks.
+ *
+ * @author barnabasmakonda@gmail.com (Barnabas Makonda)
+ */
+
+var argv = require('yargs')
+    .usage(
+      'Usage: $0 --gaeDevserverPath [path to app engine]' +
+      '--clear_datastore [whether to clear all data storage]' +
+      '--enable_sendmail [whether to send emails]' +
+      '--minify [whether to minify third-party dependencies]').argv;
+var concat = require('gulp-concat');
+var gulp = require('gulp');
+var gulpStartGae = require('./scripts/gulp-start-gae-devserver');
+var gulpUtil = require('gulp-util');
+var manifest = require('./manifest.json');
+var minifyCss = require('gulp-minify-css');
+var path = require('path');
+
+var gaeDevserverPath = argv.gaeDevserverPath;
```

Comment:
I think I found the issue: when I suggested renaming the variable here I only meant the first one, i.e.

    var gaeDevserverPath = argv.gae_devserver_path

Otherwise, if you do it like this, the calls to this script that pass in gae_devserver_path stop working, because they're passing in gaeDevserverPath instead. This results in a later script having an error because the path being passed to it is undefined.

I would recommend doing a bit of error checking in this file to ensure that the relevant arguments have been set and to raise a more informative error if they aren't.